compiler/pipes/calc-const-types.cpp

```diff
@@ -63,7 +64,7 @@ VertexPtr CalcConstTypePass::on_exit_vertex(VertexPtr v) {
       break;
     }
     default:
-      kphp_error (0, fmt_format("Unknown cnst-type for [op = {}]", v->type()));
+      kphp_error (0, fmt_format("Unknown cnst-type for [op = {}]", vk::to_underlying(v->type())));
```

Comment:
Instead of adding a function with unknown usage perspective, I'd just use `static_cast<std::to_underlying_t<decltype(v->type())>>(v->type())` here

Reply:
Disagree. 

> unknown usage
It'a a value judgment. Such a function is accepted in C++23, so I think it's useful wrapper and some programmers khow how to use it.

Using pure static cast with decltypes is too verbose and reduces readability, as for me.  I think it's a matter of taste and not really the deal for discussion.